edtLHD/src/edittree

```diff
@@ -0,0 +1,24 @@
+.
+├── main
+│   ├── java
+│   │   └── fr
+│   │       └── univtln
+│   │           └── lhd
+│   │               ├── controllers
+│   │               ├── exceptions
+│   │               ├── facade
+│   │               │   └── Schedule.java
+│   │               ├── model
```

Comment:
Why put this as a file?

Reply:
What do you mean ?

Reply:
This is a file in the commit, is it necessary?

Reply:
ah ok didn't saw it before